clang/test/CodeGenSYCL/free_function_int_header.cpp

```diff
@@ -0,0 +1,28 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -triple spir64-unknown-unknown -sycl-std=2020 -fsycl-int-header=%t.h %s -emit-llvm -o %t.ll
```

Comment:
You don't need `-emit-llvm` or `t.ll` for this test

Reply:
Ok, removed.